static/app/views/releases/detail/overview/releaseComparisonChart/index.tsx

```diff
@@ -349,207 +357,227 @@ function ReleaseComparisonChart({
-        : null,
-      diffDirection: diffCrashFreeUsers ? (diffCrashFreeUsers > 0 ? 'up' : 'down') : null,
-      diffColor: diffCrashFreeUsers
-        ? diffCrashFreeUsers > 0
-          ? 'green300'
-          : 'red300'
-        : null,
-    },
-    {
-      type: ReleaseComparisonChartType.HEALTHY_USERS,
-      role: 'children',
-      thisRelease: defined(releaseHealthyUsers)
-        ? displaySessionStatusPercent(releaseHealthyUsers)
-        : null,
-      allReleases: defined(allHealthyUsers)
-        ? displaySessionStatusPercent(allHealthyUsers)
-        : null,
-      diff: defined(diffHealthyUsers)
-        ? displaySessionStatusPercent(diffHealthyUsers)
-        : null,
-      diffDirection: diffHealthyUsers ? (diffHealthyUsers > 0 ? 'up' : 'down') : null,
-      diffColor: diffHealthyUsers ? (diffHealthyUsers > 0 ? 'green300' : 'red300') : null,
-    },
-    {
-      type: ReleaseComparisonChartType.ABNORMAL_USERS,
-      role: 'children',
-      thisRelease: defined(releaseAbnormalUsers)
-        ? displaySessionStatusPercent(releaseAbnormalUsers)
-        : null,
-      allReleases: defined(allAbnormalUsers)
-        ? displaySessionStatusPercent(allAbnormalUsers)
-        : null,
-      diff: defined(diffAbnormalUsers)
-        ? displaySessionStatusPercent(diffAbnormalUsers)
-        : null,
-      diffDirection: diffAbnormalUsers ? (diffAbnormalUsers > 0 ? 'up' : 'down') : null,
-      diffColor: diffAbnormalUsers
-        ? diffAbnormalUsers > 0
-          ? 'red300'
-          : 'green300'
-        : null,
-    },
-    {
-      type: ReleaseComparisonChartType.ERRORED_USERS,
-      role: 'children',
-      thisRelease: defined(releaseErroredUsers)
-        ? displaySessionStatusPercent(releaseErroredUsers)
-        : null,
-      allReleases: defined(allErroredUsers)
-        ? displaySessionStatusPercent(allErroredUsers)
-        : null,
-      diff: defined(diffErroredUsers)
-        ? displaySessionStatusPercent(diffErroredUsers)
-        : null,
-      diffDirection: diffErroredUsers ? (diffErroredUsers > 0 ? 'up' : 'down') : null,
-      diffColor: diffErroredUsers ? (diffErroredUsers > 0 ? 'red300' : 'green300') : null,
-    },
-    {
-      type: ReleaseComparisonChartType.CRASHED_USERS,
-      role: 'default',
-      thisRelease: defined(releaseCrashedUsers)
-        ? displaySessionStatusPercent(releaseCrashedUsers)
-        : null,
-      allReleases: defined(allCrashedUsers)
-        ? displaySessionStatusPercent(allCrashedUsers)
-        : null,
-      diff: defined(diffCrashedUsers)
-        ? displaySessionStatusPercent(diffCrashedUsers)
-        : null,
-      diffDirection: diffCrashedUsers ? (diffCrashedUsers > 0 ? 'up' : 'down') : null,
-      diffColor: diffCrashedUsers ? (diffCrashedUsers > 0 ? 'red300' : 'green300') : null,
-    },
-    {
+  if (hasHealthData) {
+    charts.push(
+      {
+        type: ReleaseComparisonChartType.CRASH_FREE_SESSIONS,
```

Comment:
This is the same, just wrapped in conditionals `hasDiscover, hasPerformance, hasHealthData`.